src/test/java/com/sda/training/pages/AuthenticationPage.java

```diff
@@ -0,0 +1,76 @@
+package com.sda.training.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class AuthenticationPage extends BasePage{
+
+    @FindBy(how = How.ID, using = "create-account_form")
+    WebElement createAccountForm;
+
+    @FindBy(how = How.ID, using = "SubmitCreate")
+    WebElement createAccountButton;
+
+    @FindBy(how = How.ID, using = "create_account_error")
+    WebElement createAccountError;
+
+    @FindBy(how = How.ID, using = "email_create")
+    WebElement emailField;
+
+    @FindBy(how = How.ID, using = "email")
+    WebElement loginEmailField;
+
+    @FindBy(how = How.ID, using = "passwd")
+    WebElement passwordField;
+
+    @FindBy(how = How.ID, using = "SubmitLogin")
+    WebElement signInButton;
+
+    @FindBy(how = How.XPATH, using = "/html/body/div/div[2]/div/div[3]/div/div[1]/ol/li")
+    WebElement alertMessage;
+
+    public void verifyIfAuthenticationPageIsOpened(){
+        new WebDriverWait(driver, 10).until(ExpectedConditions.visibilityOf(createAccountForm));
+    }
+
+    public void clickCreateAccountButton(){
+        new WebDriverWait(driver, 10).until(ExpectedConditions.visibilityOf(createAccountButton));
```

Comment:
This seems to be code duplication. Maybe extracting in BasePage class method waitUntil will be better option?
Also 10 could be extracted to constant TIMEOUT_IN_MS for example.
Code in commented method after refactor could be look like below:
```suggestion
       waitUntil(ExpectedConditions.visibilityOf(createAccountButton));
```